### PR TITLE
Adding deprecations that were recommended by DepMiner and rewritten by Guille

### PR DESCRIPTION
**Pull request by olekscode**
Those methods were present in Pharo 8 but deleted in Pharo 9.
DepMiner tool (https://github.com/olekscode/DepMiner) recommended to return them with deprecations.

Those recommendations were rejected by @Ducasse, @MarcusDenker, and @tesonep.
However, they were accepted by @guillep who also modified the generated code.

### CODE REVIEW

**Review by MarcusDenker — DISMISSED** [review]

**Comment by Ducasse** [comment]
strange that the build did not finished yet

**Comment by MarcusDenker** [comment]
Found undeclared references: #(#window #handle)
ReleaseTest(TestAsserter)>>assert:description:resumable:
ReleaseTest(TestAsserter)>>assert:description:
ReleaseTest>>testUndeclared

File: src/OSWindow-Core/OSWindow.class.st [hunk 1]
```diff
@@ -226,6 +226,13 @@ OSWindow >> getFlags [
 	^ self validHandle getFlags
 ]
 
+{ #category : #generated }
+OSWindow >> handle [
+
+	self deprecated: 'Use #sdl2Window instead'.
+	^ handle
```
Comment (MarcusDenker) [inline src/OSWindow-Core/OSWindow.class.st:233]: the ivar seems to not exist

**Comment by Ducasse** [comment]
@olekscode can you fix these so that we integrate it.

**Comment by olekscode** [comment]
I can not check out this branch from Pharo 9 anymore
It raises an error and freezes my image

So please reject this PR and I will open a new one.
There is only one method that should be accepted

**Comment by olekscode** [comment]
New PR is here: #8672

**Pull request closed without merging**